features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/resource-access-control-v2.xml.j2

```diff
@@ -832,12 +832,12 @@
         </Resource>
 
         <!-- [Organization] Admin Application Management API -->
-        <Resource context="(.*)/api/server/v1/applications/(.*)/owner(.*)" secured="true" http-method="PUT">
-            <Scopes>internal_organization_admin</Scopes>
+        <Resource context="(.*)/o/api/server/v1/applications/(.*)/owner(.*)" secured="true" http-method="PUT">
```

Comment:
Checked this file. There was no tenant level entry for 
`(.*)/api/server/v1/applications/(.*)/owner(.*)` after refactoring as here. Don't we need tenant API for that?

Reply:
fixed